File: BAEKJOON/java/2587_대표값2.java
import java.io.*;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public class Main {
    public static List<Integer> nums = new ArrayList<>();

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        input();
        solution();

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
        for (int i = 0; i < 5; i++) {
            int n = Integer.parseInt(br.readLine());
            nums.add(n);
        }
    }

    private static void solution() throws IOException {
        Collections.sort(nums);

        int sum = 0;
        for (int num : nums) {
            sum += num;
        }

        int avg = sum / 5;
        int mid = nums.get(2);

        bw.write(avg + "\n" + mid);
    }
}

File: BAEKJOON/cpp/2587_대표값2.cpp
#include <iostream>
#include <algorithm>
#include <vector>
using namespace std;

vector<int> nums;

void input() {
	int n;
	for (int i = 0; i < 5; i++) {
		cin >> n;
		nums.push_back(n);
	}
}

void solution() {
	sort(nums.begin(), nums.end());

	int sum = 0;
	for (int num : nums) {
		sum += num;
	}

	int avg = sum / 5;
	int mid = nums[2];

	cout << avg << "\n" << mid;
}

int main()
{
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	input();
	solution();

	return 0;
}